pipeline/pipeline.go

```diff
@@ -89,15 +96,17 @@ type Pipeline struct {
 
 	metricsHolder *metricsHolder
 
-	// some debugging shit
```

Comment:
To offensive? :)
Looks funny to me

Reply:
@vano144 said we should slowly get rid of this kind of stuff :-)